spark/v3.3/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestCopyOnWriteMerge.java

```diff
@@ -81,6 +83,7 @@ public synchronized void testMergeWithConcurrentTableRefresh() throws Exception
         tableName, MERGE_ISOLATION_LEVEL, "snapshot");
 
     sql("INSERT INTO TABLE %s VALUES (1, 'hr')", tableName);
+    createBranch();
```

Comment:
Do we need to create a branch since it isn't used in the rest of this test?

Reply:
missed the change of the line later to use `commitTarget`, updated